Format conversion error in coverage test

**Describe the bug**
When compiling the coverage test on RTEMS, a format conversion error is triggered:

```
/home/joe/code/cfecfs/github/apps/sample_app/unit-test/coveragetest/coveragetest_sample_app.c:152:5: error: format '%d' expects argument of type 'int', but argument 6 has type 'uint32' [-Werror=format=]
     UtAssert_True(SAMPLE_AppData.RunStatus == CFE_ES_RunStatus_APP_ERROR,
     ^
cc1: all warnings being treated as errors
```

**To Reproduce**
Build with ENABLE_UNIT_TESTS=TRUE for i686-rtems4.11 platform

**Expected behavior**
Should build clean

**System observed on:**
i686-rtems4.11 cross build

**Reporter Info**
Joseph Hickey, Vantage Systems, Inc.

--- a/unit-test/coveragetest/coveragetest_sample_app.c
+++ b/unit-test/coveragetest/coveragetest_sample_app.c
@@ -150,8 +150,8 @@ void Test_SAMPLE_AppMain(void)
      * log will show what the incorrect value was.
      */
     UtAssert_True(SAMPLE_AppData.RunStatus == CFE_ES_RunStatus_APP_ERROR,
-            "SAMPLE_AppData.RunStatus (%d) == CFE_ES_RunStatus_APP_ERROR",
-            SAMPLE_AppData.RunStatus);
+            "SAMPLE_AppData.RunStatus (%lu) == CFE_ES_RunStatus_APP_ERROR",
+            (unsigned long)SAMPLE_AppData.RunStatus);
 
 
     /*

